core/src/main/scala/akka/persistence/jdbc/testkit/internal/SchemaUtilsImpl.scala

```diff
@@ -39,8 +39,10 @@ private[jdbc] object SchemaUtilsImpl {
   @InternalApi
   private[jdbc] def dropIfExists(logger: Logger)(implicit actorSystem: ClassicActorSystemProvider): Future[Done] = {
 
+    val legacy = actorSystem.classicSystem.settings.config
+        .getString("jdbc-journal.dao") != "akka.persistence.jdbc.journal.dao.AkkaSerializerJournalDao"
```

Comment:
This test lives in multiple places, can it be centralized?

Reply:
👍